examples/lowMach/lowMach.udf

```diff
@@ -9,13 +9,34 @@
 
 static occa::kernel userQKernel;
 static occa::kernel userVpKernel;
+static occa::kernel alphavKernel;
+occa::memory o_alphav;
 
 static dfloat P_DELTA;
 static dfloat P_GAMMA;
 
 void qtl(nrs_t* nrs, dfloat time, occa::memory o_div)
 {
-  lowMach::qThermalIdealGasSingleComponent(time, o_div);
+  mesh_t *mesh = nrs->meshV;
+  platform_t* platform = platform_t::getInstance();  
+  cds_t* cds = nrs->cds;
+
+  o_alphav = platform->device.malloc(nrs->fieldOffset*sizeof(dfloat));
+   
+// option 1 , for ideal gas
+//for ideal gas, alphav =  1/T
+//qtl = 1/(rho*cp*T) * (div[k*grad[T] ] + qvol)
+// copy T to alphav
+  o_alphav.copyFrom(cds->o_S, nrs->fieldOffset * sizeof(dfloat)); 
+// take inverse of 1/T
+  platform->linAlg->ady(mesh->Nlocal,1.0,o_alphav);
+  
+// option 2, for real Gas
```

Comment:
Let's use just this option and remove the code above. Can we also rename

o_alphav => o_expansionCoeff